kitchen.yml

```diff
@@ -55,9 +61,9 @@ suites:
   - name: smoke_package_stable
     run_list: jenkins_smoke::default
     excludes:
-      - ubuntu-14.04
       - windows-2012r2
       - windows-2016
+      - widnows-2019
```

Comment:
You have a typo in the exclude name here

Reply:
nice catch